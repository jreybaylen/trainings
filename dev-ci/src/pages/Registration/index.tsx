import React from 'react'

function Registration (): JSX.Element {
    const registrationElement = (
        <div>Hello Registration!</div>
    )

    return registrationElement
}

export default Registration