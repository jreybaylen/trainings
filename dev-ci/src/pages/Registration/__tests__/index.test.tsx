import React from 'react'
import { render } from '@testing-library/react'

import Registration from '../'

describe('Registration Page', () => {
    it('should render without crashing', () => {
        const { getByText } = render(<Registration />)

        expect(getByText(/Hello Registration\!/)).toMatchSnapshot()
    })
})