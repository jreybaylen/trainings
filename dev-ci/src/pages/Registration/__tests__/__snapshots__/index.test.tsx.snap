// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Page should render without crashing 1`] = `
<div>
  Hello Registration!
</div>
`;
